### PR TITLE
add note to FAQ how to check if a file is executed using julia file.jl

### PR DESCRIPTION
**Pull request by KristofferC**
I think this is the right way to do it... cc @omus

### CODE REVIEW

**Comment by innerlee** [comment]
or execute using `./file.jl`

File: doc/src/manual/faq.md [hunk 1]
```diff
@@ -645,6 +645,13 @@ as nothing but rather a tuple of zero values.
 The empty (or "bottom") type, written as `Union{}` (an empty union type), is a type with
 no values and no subtypes (except itself). You will generally not need to use this type.
 
+
+### How do I check if a file is directly executed by Julia?
```
Comment (StefanKarpinski) [inline doc/src/manual/faq.md]: A `.jl` file is always executed by Julia so I find this working confusing. How about:

> How do I check if the current `.jl` file is being run as the main script?

**Comment by KristofferC** [comment]
Updated

File: doc/src/manual/faq.md [hunk 2]
```diff
@@ -645,6 +645,13 @@ as nothing but rather a tuple of zero values.
 The empty (or "bottom") type, written as `Union{}` (an empty union type), is a type with
 no values and no subtypes (except itself). You will generally not need to use this type.
 
+
+### How do I check if the current file is being run as the main script?
+
+When a file is run as the main script using `julia file.jl` one might want to activate extra
+functionality like command line argument handling. A way to determine that a file is run in
+such as fashion is to check if `abspath(PROGRAM_FILE) == @__FILE__` is `true`.
```
Comment (innerlee) [inline doc/src/manual/faq.md]: such a fashion

Comment (KristofferC) [reply doc/src/manual/faq.md]: Thx

**Pull request merged**